lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -490,7 +490,7 @@ def get_distribution_AIX(self):
         rc, out, err = self.module.run_command("/usr/bin/oslevel")
         data = out.split('.')
         aix_facts['distribution_major_version'] = data[0]
-        aix_facts['distribution_version'] = data[0]
+        aix_facts['distribution_version'] = '%s.%s' % (data[0], data[1])
```

Comment:
to be on the safe side and since I don't have access to test on AIX, could you make this conditional on data[1] existing? i.e `len(data) > 1`